json-guides/bulkhead.json

```diff
@@ -233,10 +234,11 @@
                 "Using the <code>@Bulkhead</code> annotation limits the number of concurrent financial advisor chat requests. After the maximum limit of concurrent chat requests to the VirtualFinancialAdvisor microservice is reached, the next chat request fails. Adding the <code>@Asynchronous</code> annotation eases this problem by allowing additional chat requests to be placed in a waiting queue.<br>",
                 "When the <code>@Bulkhead</code> annotation is used with the <code>@Asynchronous</code> annotation, access to the microservice is managed by a thread pool. This allows you to configure the maximum concurrent requests along with a waiting queue size. When a request for a financial advisor cannot be added to the waiting queue, a <code>BulkheadException</code> is thrown.<br>",
                 "Configure the <code>@Bulkhead</code> annotation, when using it with the <code>@Asynchronous</code> annotation, with the following parameters:<br>",
-                "<ul><li><b>value</b>: The maximum number of concurrent requests to the service. If the parameter is not specified, the default is <code>10</code> requests.</li><li><b>waitingTaskQueue</b>: The size of the waiting queue which holds requests that will execute at a different time. If the parameter is not specified, the default is <code>10</code> requests. This parameter for the <code>@Bulkhead</code> annotation only takes effect when the <code>@Asynchronous</code> annotation is used.</li></ul><br>"         
+                "<ul><li><b>value</b>: The maximum number of concurrent requests to the service. If the parameter is not specified, the default is <code>10</code> requests.</li><li><b>waitingTaskQueue</b>: The size of the waiting queue which holds requests that will execute at a different time. If the parameter is not specified, the default is <code>10</code> requests. This parameter for the <code>@Bulkhead</code> annotation only takes effect when the <code>@Asynchronous</code> annotation is used.</li></ul>",
+                "Modify your <code>server.xml</code> file to <a href='#enabling-microprofile-fault-tolerance'>include the fault tolerance feature</a>."         
```

Comment:
This statement seems out of place here.   How about adding it to the paragraph beginning with 'Configure the @Bulkhead annotation , when using it with the @Asynchronous annotation, with ...." as follows:

To use the @Bulkhead and @Asynchronous annotations together, you must modify your <code>server.xml</code> file to <a href='#enabling-microprofile-fault-tolerance'>include the fault tolerance feature</a>.  Then, configure the <code>@Bulkhead</code> annotation with the following parameters:
<li>value
<li>waitingTaskQueue